### ISSUE TITLE
Consecutive `prisma migrate save --experimental` output invalid Database Steps

### ISSUE THREAD

**Issue by dilizarov** [prisma/migrate#595]
## Bug description

Creating migrations through `npx prisma migrate save --experimental` consecutively without `npx prisma migrate up --experimental` produces incorrect `README.md` information. I have not verified if it also produces inaccurate DB calls, though I imagine it does.

## How to reproduce

Assume the following model in `schema.prisma` and assume it has already been migrated/persisted to the DB.

```prisma
model User {
  name String
  nickname String?
}
```

1. Add two new columns to the model

```prisma
model User {
  name String
  nickname String?
  biography String
  profilePicture String
}
```

2. Run `npx prisma migrate save --experimental`
3. Check out `README.md` of the created migration.

##Database Steps

```sql
ALTER TABLE "public"."User" ADD COLUMN "biography" text   NOT NULL ,
ADD COLUMN "profilePicture" text   NOT NULL 
```

4. Change the nickname column and make it required on the User model now

```prisma
model User {
  name String
  nickname String
  biography String
  profilePicture String
}
```

5. Run `npx prisma migrate save --experimental`
6. Check out the `README.md` for the new created migration

##Database Steps

```sql
ALTER TABLE "public"."User" ADD COLUMN "biography" text   NOT NULL ,
ADD COLUMN "profilePicture" text   NOT NULL,
ALTER COLUMN "nickname" SET NOT NULL 
```
 
This is incorrect. While it is true that `npx prisma migrate up --experimental` was not run, and both migrations need to be applied to the DB, this one migration is not and should not be responsible for adding the two new created columns.

## Expected behavior

The expected behavior is for the second migration `README.md` to have the following:

##Database Steps

```sql
ALTER COLUMN "nickname" SET NOT NULL 
```

Obviously, it should also act accordingly.

## Prisma information

The above example should suffice. If it does not, I'll share details about my actual schema as needed.

## Environment & setup

- OS: Mac OS 10.15.3
- Database: PostgreSQL
- Node.js version: 12.10.0
- Prisma version:
```
@prisma/cli          : 2.7.0
Current platform     : darwin
Query Engine         : query-engine eaade828a21d8ee3f4940f0af7da3ae0922db4df (at node_modules/@prisma/cli/query-engine-darwin)
Migration Engine     : migration-engine-cli eaade828a21d8ee3f4940f0af7da3ae0922db4df (at node_modules/@prisma/cli/migration-engine-darwin)
Introspection Engine : introspection-core eaade828a21d8ee3f4940f0af7da3ae0922db4df (at node_modules/@prisma/cli/introspection-engine-darwin)
Format Binary        : prisma-fmt eaade828a21d8ee3f4940f0af7da3ae0922db4df (at node_modules/@prisma/cli/prisma-fmt-darwin)
Studio               : 0.287.0
```

**Comment by albertoperdomo** [comment]
Can you please repro @pantharshit00 ?

**Comment by pantharshit00** [comment]
I can reproduce this with `2.9.0-dev.35`. Thanks for the detailed steps cc @tomhoule

**Comment by tomhoule** [comment]
Thanks for taking the time to report this issue!

We've recently released a Prisma Migrate Preview ([2.1.3.0 release notes]([https://github.com/prisma/prisma/releases/tag/2.13.0](https://github.com/prisma/prisma/releases/tag/2.13.0))), which has quite a few changes compared to the previous experimental version. We believe this issue is no longer relevant in this new version, so we are closing this.

We would encourage you to try out Prisma Migrate Preview. If you come across this or any other issue in the preview version, please feel free to open a [new issue in prisma/prisma]([https://github.com/prisma/prisma/issues/new/choose](https://github.com/prisma/prisma/issues/new/choose)).
For general feedback on Prisma Migrate Preview, feel free to chime in [on this issue]([https://github.com/prisma/prisma/issues/4531](https://github.com/prisma/prisma/issues/4531)).
